src/Symfony/Bundle/FrameworkBundle/Resources/config/mime_type.xml

```diff
@@ -7,6 +7,10 @@
     <services>
         <defaults public="false" />
 
-        <service id="mime_types" class="Symfony\Component\Mime\MimeTypes" public="true" />
+        <service id="mime_types" class="Symfony\Component\Mime\MimeTypes" public="true">
```

Comment:
Not sure if we really still want to have the service `public` now. I don't see why that's still necessary.

Reply:
:+1: to remove `public="true"`

Reply:
here we go